src/style.css

```diff
@@ -0,0 +1,121 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.container-big-bar {
+  position: relative;
+  display: block;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  height: 68px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+}
+
+.big-search-bar {
+  position: absolute;
+  box-sizing: border-box;
+  left: 62px;
+  border: none;
+  top: 23px;
+  height: 22px;
```

Comment:
Correct [property order](https://9elements.com/css-rule-order/). Fix it everywhere